### DISCUSSION TITLE
Looking for  conductor workflow definition creation in UI

### DISCUSSION THREAD

**Discussion by praveeng-nair** [Netflix/conductor#2892] [General]
Hi Team,
one of the most painful task in conductor is making some updates on the workflows since sometimes they are very big ones and a bit complex. is there any UI from where we can create the workflow definitions or this capability added in any later versions of conductor. changing big json takes a lot time to figure where the actual change has to go....
@aravindanr @apanicker-nflx @v1r3n

**Comment by apanicker-nflx** [comment]
@praveeng-nair This ability was introduced in the UI in [v3.6.0](https://github.com/Netflix/conductor/releases/tag/v3.6.0). Please note that this is the first iteration of mutability within the UI and more enhancements are upcoming.

**Reply by praveeng-nair** [reply]
Thanks @apanicker-nflx
